pytorch_blade/tests/tensorrt/test_onnx.py

```diff
@@ -67,5 +68,38 @@ def forward(self, x):
         with io.BytesIO() as f:
             onnx.save(onnx_proto, f)
 
+    @unittest.skipIf(utils.torch_version_number() <= utils.parse_version("1.7.1"), 'failed')
+    def test_fuse_conv_bn(self):
+        class Net(nn.Module):
+            def __init__(self):
+                super(Net, self).__init__()
+                self.conv = nn.Conv2d(3, 6, 5)
+                self.bn = nn.BatchNorm2d(6)
+
+            def forward(self, x):
+                x = F.relu(self.bn(self.conv(x)))
+                return x
+
+        script_module = torch.jit.script(Net()).eval()
+        inputs = torch.ones([1, 3, 20, 20])
+        script_module = exporter.export(script_module, model_inputs=(inputs,))
+        script_module = pass_manager._optimize_common(script_module._c)
+        onnx_graph, _ = pass_manager._jit_pass_lower_to_onnx(script_module.forward.graph)
+        print(onnx_graph)
```

Comment:
Nit: should not print directly in the ut

Reply:
Done.